juri/styles.css

```diff
@@ -0,0 +1,179 @@
+html{
+  box-sizing: border-box;
```

Comment:
You can also put `font-family: var(--main-font-family);` cause there is only 1 fonts used in the document